src/ui/components/utils/GlobalErrorBoundary.tsx

```diff
@@ -15,7 +15,7 @@ export class GlobalErrorBoundary extends React.Component<
     if (hasError && error && info) {
       return (
         <div style={{ padding: 10 }}>
-          <h1>NextEditor: Recorvery Mode</h1>
+          <h1>NextEditor: Recovery Mode</h1>
```

Comment:
@t8 Could you also rename `NextEditor:` to `Gitopia:`
This is the remnant of the repo we started with.